resources/droplet_resource.go

```diff
@@ -35,3 +39,77 @@ type DropletBuildpack struct {
 	// Version is the version of the detected buildpack.
 	Version string `json:"version"`
 }
+
+func (d Droplet) MarshallJSON() ([]byte, error) {
+	type Data struct {
+		GUID string `json:"guid,omitempty"`
+	}
+
+	type RelationshipData struct {
+		Data Data `json:"data,omitempty"`
+	}
+
+	type Relationships struct {
+		App RelationshipData `json:"app,omitempty"`
+	}
```

Comment:
Any particular reason for the difference between these structs and the ones for the unmarshal (named structs vs anonymous structs)? My suggestion is if we are keeping these inside functions personally I would prefer to have them anonymous.

Reply:
@gmllt wanted to follow up. Do you have any feedback on this?